.babelrc

```diff
@@ -1,50 +1,88 @@
 {
   "env": {
-    "test": {
+    "development": {
       "presets": [
-        ["env", {"targets": { "node": "current" }}],
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
+          }
+        ],
         "react",
         "stage-2"
-      ],
-      "plugins": [
-        "babel-plugin-rewire",
-        ["module-resolver", {
-          "alias": {
-            "preact-compat": "react"
+      ]
+    },
+    "production": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
           }
-        }]
+        ],
+        "react",
+        "stage-2"
       ]
     },
-    "development": {
+    "umd": {
```

Comment:
Why do we need the `umd` since it's the same as `es`?

Reply:
They're not the same: the ES bundle use the `inline-replace-variables` plugin. However, `es` and `cjs` are the same.